src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -14,10 +13,21 @@ import { Providers } from '../../Providers';
 import { ProviderState } from '../../providers/IProvider';
 import '../../styles/fabric-icon-font';
 import { debounce } from '../../utils/Utils';
-import '../mgt-person/mgt-person';
+import { IDynamicPerson } from '../mgt-person/mgt-person';
 import { MgtTemplatedComponent } from '../templatedComponent';
 import { styles } from './mgt-people-picker-css';
 
+/**
+ * An interface used to mark an object as 'selected',
+ * so it can be rendered differently.
+ *
+ * @interface ISelectable
+ */
+interface ISelectable {
+  // tslint:disable-next-line: completed-docs
+  isSelected: boolean;
```

Comment:
Should this be `isFocused` based on how it's used?

Reply:
Good point. `isFocused` makes more sense.